packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -80,6 +85,17 @@ export interface IPluginLedgerConnectorBesuOptions
   logLevel?: LogLevelDesc;
 }
 
+// export interface Log{
```

Comment:
@awadhana Forgot to mention this today: Please remove this leftover code (the commented out interface definition)